content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Change Notes
+description: Istio 1.6 release notes.
+weight: 10
+---
+
+## Traffic Management
+
+- ***Added*** the new [Workload Entry](/docs/reference/config/networking/workload-entry/) resource. This will allow easier configuration for non-Kubernetes workloads to join the mesh.
+- ***Added*** configuration for gateway topology. This addresses sanitizing and providing useful [X-Forward-For headers](https://github.com/istio/istio/issues/7679).
+- ***Added*** experimental support for the [Kubernetes Service APIs](https://github.com/kubernetes-sigs/service-apis/).
+- ***Added*** support for using `appProtocol` to select the [protocol for a port](/docs/ops/configuration/traffic-management/protocol-selection/) introduced in Kubernetes 1.18.
+- ***Improved*** support for [Kubernetes ingress](/docs/tasks/traffic-management/ingress/kubernetes-ingress/), adding support for reading certificates from Secrets, `pathType`, and `IngressClass`.
+- ***Added*** a new `proxy.istio.io/config` annotation to override proxy configuration per pod.
+- ***Removed*** most configuration flags and environment variables for the proxy. These are now read directly from the mesh configuration.
+- ***Changed*** the proxy readiness probe to port 15021.
+- ***Fixed*** a [bug](https://github.com/istio/istio/issues/16458) blocking external HTTPS/TCP traffic in some cases.
+- TODO: add a statement around file mounted gateway removed and impact to users.
+- TODO: add a statement around networking v1alpha3 API. Will it be removed in 1.6 or announce deprecation?
```

Comment:
For booking purposes, from @linsun:
> I think @howardjohn said no change here, no plan for deprecation either. if others agree (@louiscryan @rshriram @costinm @nrjpoddar ) ok to not mention it here."

Reply:
lets not mention

Reply:
removed

Reply:
```suggestion
- Networking APIs have graduated from v1alpha3 to beta.
```

Reply:
How about announce this?  I've got multiple folks complain to me that we never announce them move to beta in 1.5.